src/components/menus/directory-tree-contextual-menu.js

```diff
@@ -178,6 +329,16 @@ const DirectoryTreeContextualMenu = (props) => {
                         },
                         icon: <DeleteIcon fontSize="small" />,
                     },
+                    {
+                        messageDescriptorId: 'paste',
+                        callback: () => {
+                            pasteElement(
+                                directory?.elementUuid,
+                                selectionForCopy
+                            );
+                        },
+                        icon: <ContentPasteIcon fontSize="small" />,
```

Comment:
I reckon pasting should be eligible on public folders owned by other users, the features under `isAllowed()` are only available for items in one own folders

Reply:
yes will fix it